Description: Set the proper path to javacc.jar on Debian for the command line tools
Forwarded: not-needed
--- a/bin/javacc
+++ b/bin/javacc
@@ -1,9 +1,5 @@
 #!/bin/sh
-JAR="`dirname $0`/lib/javacc.jar"
-
-case "`uname`" in
-     CYGWIN*) JAR="`cygpath --windows -- "$JAR"`" ;;
-esac
+JAR=/usr/share/java/javacc.jar
 
 java -classpath "$JAR" javacc "$@"
 
--- a/bin/jjdoc
+++ b/bin/jjdoc
@@ -1,9 +1,5 @@
 #!/bin/sh
-JAR="`dirname $0`/lib/javacc.jar"
-
-case "`uname`" in
-     CYGWIN*) JAR="`cygpath --windows -- "$JAR"`" ;;
-esac
+JAR=/usr/share/java/javacc.jar
 
 java -classpath "$JAR" jjdoc "$@"
 
--- a/bin/jjtree
+++ b/bin/jjtree
@@ -1,9 +1,5 @@
 #!/bin/sh
-JAR="`dirname $0`/lib/javacc.jar"
-
-case "`uname`" in
-     CYGWIN*) JAR="`cygpath --windows -- "$JAR"`" ;;
-esac
+JAR=/usr/share/java/javacc.jar
 
 java -classpath "$JAR" jjtree "$@"
 
